src/locales/es/translations/labs/index.ts

```diff
@@ -0,0 +1,30 @@
+export default {
+  labs: {
+    label: 'Pruebas de laboratorio',
```

Comment:
I may leave a comment on a couple of these to link you to the English translation files in `shared/locales` to point you to any incompleted fields :)
https://github.com/HospitalRun/hospitalrun-frontend/blob/master/src/shared/locales/enUs/translations/labs/index.ts